src/Avalonia.Controls/Converters/MarginMultiplierConverter.cs

```diff
@@ -18,10 +18,24 @@ public class MarginMultiplierConverter : IValueConverter
 
         public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
```

Comment:
I would expect this class to be called "ThicknessMultiplierConverter", but it is public...